go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/mattn/go-sqlite3 v2.0.3+incompatible
 	github.com/monarko/fhirgo v0.0.0-20200616214506-ca0a03fb1f7a
 	github.com/nuts-foundation/go-did v0.3.0
-	github.com/nuts-foundation/nuts-node v1.0.1-0.20220505113658-c91d8ad758f6
+	github.com/nuts-foundation/nuts-node v1.1.0
```

Comment:
This is probably not what you want to do here. You have to use the commit sha, otherwise you will use code back from 8 months ago.

Reply:
Indeed, reverted.